Doc/howto/enum.rst

```diff
@@ -1149,6 +1154,8 @@ the following are true:
 
     >>> (Color.RED | Color.GREEN).name
     'RED|GREEN'
+    >>> (Perm.R & Perm.W).name is None                   # effectively `Perm(0)`
```

Comment:
```suggestion
    >>> (Perm.R & Perm.W).name is None  # effectively Perm(0)
```

I don't think we need that much space (unless it is perhaps aligned with something I don't see?) In addition, I wouldn't use single-quote references in the doctest. Finally, maybe you could actually include the Perm snippet here only, and not at the beginning (you are only using it for this specific example).

Reply:
I think it was mostly to move it out of the way.